fapolicy_analyzer/tests/reducers/test_daemon_reducer.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+from unittest.mock import MagicMock
+from ui.reducers.daemon_reducer import (
+    handle_received_daemon_start,
+    # handle_received_daemon_stop,
+    # handle_received_daemon_reload,
+    # handle_received_daemon_status,
+    # handle_received_daemon_status_update,
+)
+
+from ui.actions import DaemonState
+
+
+@pytest.fixture
+def initial_state():
+    return DaemonState(error=None, status=False, handle=None)
+
+
+def test_handle_received_daemon_start(initial_state):
+    mockDaemon = MagicMock()
+    ret = handle_received_daemon_start(initial_state, mockDaemon)
+    print(ret)
```

Comment:
Remove or convert to a `log.debug` statement.

Reply:
This file should have been removed.